amx/server/syscalls.lua

```diff
@@ -790,12 +792,8 @@ g_SAMPSyscallPrototypes = {
 	-- network stats
 	NetStats_BytesReceived = {'p'},
 	NetStats_BytesSent = {'p'},
-	NetStats_ConnectionStatus = {'p'},
 	NetStats_GetConnectedTime = {'p'},
 	NetStats_GetIpPort = {'p', 'r', 'i'},
-	NetStats_MessagesReceived = {'p'},
-	NetStats_MessagesRecvPerSecond = {'p'},
-	NetStats_MessagesSent = {'p'},

```

Comment:
I will state this once again, please let's not move this around, it makes it more difficult to catch any potential issues and overall simply makes the PR noisier. 😊

Reply:
It was just the last couple of functions which wasn't moved to dummy section, so now there are no other left, fortunately. I really try to make the picture clear for those who might come in the future and look directly at the code to see what was still unimplemented (not all of this is reflected in Issues, so an honest distinction in the code itself seemed to me an effective move for future contributions).

Reply:
I understand, but it would be better to still not move this around in future commits. If certain methods are unimplemented it would be better to open an issue for it, that way there's no need to dig through the code base.